sabitou_clients/lib/screens/profile/profile_view.dart

```diff
@@ -0,0 +1,696 @@
+                  ),
+                  Text(
+                    isAdmin
+                        ? AppInternationalizationService.to.adminManager
+                        : AppInternationalizationService.to.inventoryManager,
+                  ),
+                ],
+              ),
+              if (isAdmin)
+                IconButton(
+                  onPressed: () {},
+                  icon: Image.asset(
+                    'assets/images/Person_edit.png',
+                    width: 35.0,
+                    height: 35.0,
+                  ),
+                ),
+            ],
+          ),
+          const _Spacer(),
+          LayoutBuilder(
+            builder: (context, constraints) {
+              return Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  _UserAvatar(
+                    imageUrl: user.imageUrl,
+                    fullName: '${user.firstName} ${user.lastName}',
+                    isOnline: true,
+                  ),
+                  const SizedBox(width: 20.0),
+                  if (isAdmin)
+                    const Expanded(
+                      child: _AvatarEditButtons(),
+                    ),
+                ],
+              );
+            },
+          ),
+          const SizedBox(height: 16.0),
+          _NameFields(firstName: user.firstName, lastName: user.lastName),
+          const _Spacer(),
+          Text(
+            AppInternationalizationService.to.contactEmail,
+            style: const TextStyle(fontWeight: FontWeight.bold),
+          ),
+          Text(
+            AppInternationalizationService.to.manageYourAccountEmailAddress,
+          ),
+          const SizedBox(height: 16.0),
+          _EmailPasswordFields(
+            email: user.email,
+            password: user.password,
+          ),
+          const _Spacer(),
+          _BusinessSection(business: user.business, isAdmin: isAdmin),
+          const SizedBox(height: 10.0),
+          _StoreSection(store: user.store, isAdmin: isAdmin),
+          const SizedBox(height: 16.0),
+          Text(
+            AppInternationalizationService.to.accountSecurity,
+            style: const TextStyle(fontWeight: FontWeight.bold),
+          ),
+          Text(
+            AppInternationalizationService.to.manageYourAccountSecurity,
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _UserAvatar extends StatelessWidget {
+  final String fullName;
+  final String imageUrl;
+  final bool isOnline;
+
+  const _UserAvatar({
+    required this.imageUrl,
+    required this.fullName,
+    required this.isOnline,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        SbContainer(
+          height: 100.0,
+          width: 100.0,
+          borderRadius: const BorderRadius.all(Radius.circular(100)),
+          child: Badge(
```

Comment:
I thought it was used here so that the dot to indicate online or offline could display over the profile picture? Just like it was done in the "base_page.dart".

Reply:
The profile is integrated into the base page, so that the header displaying the search bar and user profile is always shown on top.

Reply:
I don't think I actually understand what you are saying Sir. This class is used to build the user picture that is on the profile view page, that is what's next to those edit buttons. 
I'm probably misunderstanding what you are saying.